helpers/http_client.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+import (
+	"crypto/tls"
+	"net/http"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	httptransport "github.com/go-openapi/runtime/client"
+)
+
+func CreateHttpClient(insecureSkipVerify bool, tlsTrustStore string) *http.Client {
```

Comment:
Since this is a different package now, maybe we should have this function return a error, and log it in the `main.go`. This way we can add error checking in the new tests :)